index.html

```diff
@@ -117,7 +117,7 @@ <h2>Project name goes here</h2>
           
         <div class="group-4-5">
           <div class="project-container project-4">
-            <img src="images/project-4.jpg" alt="project-4 image">
+            <img src="images/project-4.jpg" alt="macbook pro 16">
 
             <div class="single-project-info">
               <h2>Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.